Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN make yarn
 COPY ./ui/ ./ui/
 RUN make build-ui
 
-FROM golang:1.21.5-bookworm@sha256:1415bb0b25d3bffc0a44dcf9851c20a9f8bbe558095221d931f2e4a4cc3596eb as build
+FROM golang:1.21.8-bookworm@sha256:ac14cc827536ef1a124cd2f7a03178c3335c1db8ad3807e7fdd57f74096abfa0 as build
```

Comment:
I think this is the amd64 image, rather than the multi-platform manifest.

Although I think it shouldn't matter, as these dockerfiles aren't used for the release builds? (Is that right?)

Reply:
right; I backported the script to check https://github.com/pomerium/pomerium/pull/5028 and updated images

Reply:
OK thanks!